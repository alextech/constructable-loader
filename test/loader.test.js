import {
  compile,
  getCompiler,
  getModuleSource,
} from './helpers/index'

jest.setTimeout(10000);

global.CSSStyleSheet = {
  replaceSync: function (style) {

  }
}

describe('loader', () => {
  it('should work', async () => {
    const compiler = getCompiler('./basic.js');
    const stats = await compile(compiler);

    expect(getModuleSource('./basic.css', stats)).toMatchSnapshot('module');
  });
});