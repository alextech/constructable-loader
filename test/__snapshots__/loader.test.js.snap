// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader should work: module 1`] = `
"const styleSheet = new CSSStyleSheet(); styleSheet.replaceSync(\`.panel {
    background-color: orange;
    border-style: solid;
}\`); export default styleSheet;"
`;
